eng/Signing.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <ItemGroup>
+    <ItemsToSign Include="$(SparkPackagesDir)/**/*.nupkg" />
+    <ItemsToSign Include="$(SparkPackagesDir)/**/*.snupkg" />
+    
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.dll" />
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.Worker.exe" />
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.Worker.dll" />
```

Comment:
For Linux, the assembly can be just `Microsoft.Spark.Worker`. Do we need to sign it? (I vaguely remember signing only for Windows assemblies).

Reply:
We don't need to sign the assemblies on Linux, since there is no signing validation. Also - typically Linux binaries are built on a Linux machine, where signing is not possible.

From my understanding, the things that need to be signed are:

- Binaries executed on Windows machines, along with any Windows installer files, i.e. msi.
- nupkg files. 
- vsix files, but we don't make any of those.
- macOS installer files, i.e. pkg. but we don't make any of those.

Reply:
Got it, thanks.